scripts/setup.sh

```diff
@@ -13,11 +13,20 @@
 # Move file from script folder to .git/hooks folder
 cp scripts/pre-push.sh .git/hooks/pre-push
 
+# Create a folder where all the set up files will be downloaded
+mkdir -p ../oppia-android-tools
+cd ../oppia-android-tools
+
 # Download ktlint
 KTLINT="0.37.1"
 echo Using Ktlint $KTLINT
-mkdir -p ../oppia-android-tools
-cd ../oppia-android-tools
 curl -sSLOC - https://github.com/pinterest/ktlint/releases/download/$KTLINT/ktlint
 chmod a+x ktlint
 echo Ktlint file downloaded
+
+# Download buf
+BUFVERSION="v0.15.0"
+echo Using Bufbuild version $BUFVERSION
+curl -sSLOC - https://github.com/bufbuild/buf/releases/download/$BUFVERSION/buf-$(uname -s)-$(uname -m)
```

Comment:
Will uname work correctly for MinGW/Cygwin environments on Windows?

Reply:
When we download the buf-lint file using "uname -m" and "uname -s" in Windows 10, it searches for file starting with "buf-MINGW64***". Since this file is not to be found on the repo link, it creates a file with the name "buf-MINGW64***" and the content of the file is "Not found"

Reply:
@Srikanth-Kb 
 > Apparently, for a file to be made as executable on windows, its content must start with #!  - https://stackoverflow.com/questions/25730041/updating-file-permissions-with-git-bash-on-windows-7 
when i checked the contents for ktlint, the first line is "#!/bin/sh"
that is not the case for buf-lint